docs/sample.rst

```diff
@@ -0,0 +1,6 @@
+
+Sample
+======
+
+.. autoclass:: pyrefinebio.Sample
+   :members:
```

Comment:
Are there any levers that Spinx gives us to pull to change the formatting?

![image](https://user-images.githubusercontent.com/4026833/104374212-89c15d80-54ef-11eb-8df3-b689c0cae584.png)

is a pretty big block that is not easy to read.